### DISCUSSION TITLE
English Manual On "What Is Latteart ?

### DISCUSSION THREAD

**Discussion by GochaLearningSpace** [latteart-org/latteart#450] [Q&A] [answered]
Gocha@Willings.

Do we have English version of the 1st step document for new developers ? 
I could find that in Japanese in below but could not find the English version, so it's not so obvious for developers who don't use English about what kind of test tool can handle.

![image](https://user-images.githubusercontent.com/6176526/189421362-c2fe618c-75c5-4279-adba-67b69b892cb6.png)

https://github.com/latteart-org/latteart/blob/main/README_ja.md

**Comment by masaki-tajima** [comment, marked as answer]
We have documentation in Japanese, but not yet in English. We have documentation for users, but not yet for developers.

Here are all the manuals we have.
- First setup manual (English)
  How to build: https://github.com/latteart-org/latteart/blob/main/README.md
- Tutorial documentation (Japanese only)
  How to install and start:  https://github.com/latteart-org/latteart/blob/main/docs/startup/pc.md
  How to capture tests: https://github.com/latteart-org/latteart/blob/main/docs/tutorial/capture/tutorial-capture.md
  How to test with LatteArt: https://github.com/latteart-org/latteart/blob/main/docs/tutorial/management/tutorial-management.md
  Details of Capturing Tool: https://github.com/latteart-org/latteart/blob/main/docs/manual/capture/manual-capture.md
  Details of Management Tool: https://github.com/latteart-org/latteart/blob/main/docs/manual/management/manual-management.md
  Details of Automatic Test Script Generation: https://github.com/latteart-org/latteart/blob/main/docs/manual/common/test-script-generation.md
